src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -361,7 +361,19 @@ def callback(f2):
             self.assertFalse(master.exists(jobOnMaster.jobStoreID))
             # TODO: Who deletes the shared files?
 
-
+        def testBatchCreate(self):
+            master = self.master
+            masterRequirements = dict(memory=12, cores=34, disk=35, preemptable=True)
+            jobGraphs = []
+            for i in range(10):
```

Comment:
It's better to do 50 or 100, IMO, to test the batch size limits. SimpleDB limit is 25, not sure about other job stores. Better to go over than to be too conservative.